src/cmakeTaskProvider.ts

```diff
@@ -161,6 +161,15 @@ export class CMakeTaskProvider implements vscode.TaskProvider {
         }
         return undefined;
     }
+    public async cloneTaskAndAddTargets(task: CMakeTask, targets: string[]): Promise<CMakeTask | undefined> {
```

Comment:
I can't see where this function is being called. Did you mean to call it in `_doCMakeBuild`?

Reply:
Nope. It's remaining of my attempt to clone target and inject list of target into it.
Forgot to remove it from here.
Will do it.

Reply:
Removed in latest commit